io/plugins/eu.esdihumboldt.hale.io.xls.test/META-INF/MANIFEST.MF

```diff
@@ -12,14 +12,16 @@ Require-Bundle: org.junit,
  eu.esdihumboldt.hale.common.instance;bundle-version="2.9.0",
  eu.esdihumboldt.hale.common.schema;bundle-version="2.9.0",
  eu.esdihumboldt.hale.io.xls;bundle-version="2.9.0",
- eu.esdihumboldt.hale.common.schema.persist;bundle-version="2.9.0"
+ eu.esdihumboldt.hale.common.schema.persist;bundle-version="2.9.0",
+ groovy-all;bundle-version="2.4.14"
```

Comment:
The bundle that should be included as dependency for Groovy should be `groovy` instead of `groovy-all`.